encode.go

```diff
@@ -324,25 +323,26 @@ func formatTs(t time.Time) (b []byte) {
 		bc = true
 	}
 	b = []byte(t.Format(time.RFC3339Nano))
-	if bc {
-		b = append(b, "  BC"...)
-	}
 
 	_, offset := t.Zone()
 	offset = offset % 60
-	if offset == 0 {
-		return b
-	}
+	if offset != 0 {
 
-	if offset < 0 {
-		offset = -offset
+		if offset < 0 {
+			offset = -offset
+		}
+
+		b = append(b, ':')
+		if offset < 10 {
+			b = append(b, '0')
+		}
+		b = strconv.AppendInt(b, int64(offset), 10)
```

Comment:
I see that you're moving existing code, but I'm curious why we convert offset to int64 here?